src/evict/evict_conn.c

```diff
@@ -290,7 +294,9 @@ __wt_evict_create(WT_SESSION_IMPL *session, const char *cfg[])
 
 /*
  * __wt_evict_destroy --
- *     Destroy Eviction
+ *     This function releases all memory and locks related to eviction. It ensures the eviction
+ *     system is properly destroyed, It must be called exactly once during `WT_CONNECTION::close`,
+ *     and must be called after all the eviction threads are destroyed.
```

Comment:
> must be called after all the eviction threads are destroyed

Ok, but what does that mean? is "destroy eviction threads" another method, what does that?

Reply:
This is referring to `__wt_evict_threads_destroy` which joins the threads. We're trying to say "don't delete the underlying data while threads are still using them"

Reply:
Yes @jbergler, if we wish to call this function, it has to happen after the `__wt_evict_threads_destroy` is called.

Reply:
Would it make sense to call this out, 
```suggestion
 *     and must be called after all the eviction threads are destroyed (via `__wt_evict_threads_destroy`).
```

Reply:
updated in [11cba66](https://github.com/wiredtiger/wiredtiger/pull/11096/commits/11cba663c1856e3ede7e5dbdfae661ad04de53a6)